Engine/Actions/EditorActionAddComponent.cpp

```diff
@@ -0,0 +1,44 @@
+#include "EditorActionAddComponent.h"
+#include "Main/GameObject.h"
+#include "Module/ModuleLight.h"
+#include "Main/Application.h"
+
+EditorActionAddComponent::EditorActionAddComponent(Component* comp)
```

Comment:
As a recommendation you can instantiate this as ``EditorActionAddComponent::EditorActionAddComponent(Component* comp) : component(comp){}``. Otherwise if the type you are assigning has a default constructor, two initialization will happen, one before the brackets and one after the brackets.

Reply:
You are right! Thank you!